web/app.js

```diff
@@ -176,6 +176,7 @@ var PDFViewerApplication = {
   preferencePdfBugEnabled: false,
   preferenceShowPreviousViewOnLoad: true,
   preferenceDefaultZoomValue: '',
+  preferenceEnhanceTextSelection: true,
```

Comment:
I'd assume that we want to make this controllable through a preference!?

If so, it needs to be added to https://github.com/mozilla/pdf.js/blob/master/web/default_preferences.json, and then fetched with the other ones as part of the `initialize` function, see https://github.com/mozilla/pdf.js/blob/master/web/app.js#L308.